client/src/main/views/pointingRoom/components/mainContent/ParticipantList.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Participant from './Participant';
+
+const ParticipantList = () => {
+	const participants = [
```

Comment:
Move this to a generate function inside of client/src/main/utils/mock-data/participants
(mock-data is a new folder, participants.js is the file name)

Reply:
make a prop for this component 'participants', The parent should pass the result of the new function I said above into the new prop and use that as the list you map out.